core/src/main/res/values/colors.xml

```diff
@@ -28,4 +28,9 @@
     <color name="master_switch_background_light">#DDDDDD</color>
     <color name="master_switch_background_dark">#191919</color>
 
+    <!-- filter dialog -->
+    <color name="color_custom_dialog_off">#FFFFFF</color>
+    <color name="color_custom_dialog_on">#3D8BFF</color>
+    <color name="color_custom_dialog_border">#3D8BFF</color>
```

Comment:
Please use `accent_light` and `background_light` instead of creating new colors. Having them duplicated will likely lead to inconsistent colors throughout the app. This is currently a problem in 1.8.1 (up to 3 different shades of blue on one screen) and I want to prevent that from happening again.